src/index.html

```diff
@@ -1,17 +1,197 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
+    </meta>
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      method="post"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    >
+      <fieldset class="container">
+        <legend>Personal information</legend>
+
+        <label class="box">
+          Surname:
+
+          <input
+            type="text"
+            id="surname"
+            name="surname"
+            autocomplete="off"
+            required
+          >
+        </label>
+        <label class="box">
+          Name:
+
+          <input
```

Comment:
```suggestion
        <label class="box">
          Name
          <input
```
No need to add blank line between parent and child elements

Fix all such cases